Dockerfile.centos7

```
@@ -53,6 +53,7 @@ RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
 # we need to build our own patchelf
 WORKDIR /temp-patchelf
 RUN git clone https://github.com/NixOS/patchelf.git . \
+    && git checkout 0.17.0 \
```

This will do it in one command and will pull only necessary files, so it should be faster.

```suggestion
RUN git clone -b 0.17.0 --single-branch https://github.com/NixOS/patchelf.git . \
```